recipes/openmpi/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 2.8.12)
```

```suggestion
cmake_minimum_required(VERSION 3.1)
```
The file test_package CMakeLists.txt should require CMake 3.1.
See https://github.com/conan-io/conan-center-index/blob/master/docs/error_knowledge_base.md#kb-h048-cmake-version-required